app/models/certificate_in_use_event.rb

```diff
@@ -1,8 +1,29 @@
+require 'notifications/client'
+require 'auth/authentication_backend'
+require 'notify/cert_status_notifications'
 class CertificateInUseEvent < AggregatedEvent
+  include AuthenticationBackend
+  include CertStatusNotifications
   belongs_to_aggregate :certificate
   data_attributes :in_use_at
+  after_create_commit :notification_service_team_members
 
   def attributes_to_apply
     { in_use_at: Time.now }
   end
+
+  def notification_service_team_members
+    recipients = team_recipients(certificate.component.team.team_alias)
+    Rails.logger.error("No recipients found for #{certificate.component.team.name}!") if recipients.empty?
+    mail_client = Notifications::Client.new(Rails.configuration.notify_key)
+
+    recipients.each { |email|
+      send_notification_email(mail_client: mail_client, certificate: certificate, email_address: email)
+    }
+
+    CertificateNotificationSentEvent.create(certificate: certificate)
+  rescue Notifications::Client::RequestError => e
+    CertificateNotificationSentEvent.create(certificate: certificate, notification_sent: false)
```

Comment:
as discussed, I don't think this event should be happening.. there can be multiple emails being sent and if one fails, it will set the `notification_sent` to false even if previous x emails were sent successfully...

Reply:
I thought it is useful to have an event here since we are just logging the mail client error  otherwise. We can get the certificate and the team, but I understand it seems it failed outrightly.